CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Unreleased
+- On macOS, fix application termination on `ControlFlow::Exit`
```

Comment:
Could you add a space between this line and `Unreleased` so this keeps consistent formatting?